apps/web/components/AddressFormNew/AddressFormNew.vue

```diff
@@ -98,11 +99,12 @@
 </template>
 <script setup lang="ts">
 import { type Address, AddressType } from '@plentymarkets/shop-api';
+import { ref, defineExpose } from 'vue';
```

Comment:
Use Nuxt auto-import feature, as that's apparently one of the reasons for 'Nuxt instance unavailable' error.
That means we dont need an import file, please remove it, unless Nuxt doesn't do auto-import (probably for defineExpose)

Reply:
when using script setup we need to use defineExpose for calling methods from partent to child.
Removed ref import from "vue". good catch.